### PR TITLE
Module install from root

### PR DESCRIPTION
**Pull request by nonoakij**
## Summary

I modified the command to execute `npm ci`, which installs the modules according to package-lock.json.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -7313,9 +7633,162 @@
...
+      ],
+      "engines": {
+        "node": ">= 10.0.0"
+      },
+      "funding": {
+        "type": "opencollective",
+        "url": "https://opencollective.com/parcel"
+      }
+    },
+    "node_modules/@parcel/watcher-linux-x64-glibc": {
```
Comment (nonoakij) [inline package-lock.json:7712]: We needed this package.
The build was failing because this was not written in the package-lock.

File: packages/linter/tsconfig.json [hunk 2]
```diff
@@ -66,7 +66,7 @@
     // "emitDecoratorMetadata": true,               /* Enables experimental support for emitting type metadata for decorators. */
 
     /* Advanced Options */
-    // "skipLibCheck": true,                           /* Skip type checking of declaration files. */
+    "skipLibCheck": true,                           /* Skip type checking of declaration files. */
```
Comment (h8570rg) [inline packages/linter/tsconfig.json:69]: Why is this necessary?

**Review by h8570rg — APPROVED** [review]
